.formatter.exs

```diff
@@ -3,5 +3,5 @@
     "{lib,spec,test,c_src,config}/**/*.{ex,exs}",
     "*.exs"
   ],
-  import_deps: [:membrane_core, :espec, :bundlex, :unifex]
+  import_deps: [:membrane_core, :bundlex, :unifex]
```

Comment:
One more place where `espec` shouldn't be removed for now - however it seems to not break, as .formatter.exs in  `membrane_core` has a workaround from the times before `.formatter.exs` being present in `espec`. That's ancient and remembers the introduction of formatter in Elixr 1.6 xD

Reply:
Turns out espec is really annoying with our new CI pipeline. For now I just added it to `:dev` environment because frankly I have no idea how else to get it to work and I don't have the time to rewrite them to ExUnit